### PR TITLE
[docs] Add starter documents 

### PR DESCRIPTION
**Pull request by dayo09**
This commit adds an introductory document for this project.

ONE-vscode-DCO-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

### CODE REVIEW

**Comment by dayo09** [comment]
### Software Stack

image

File: docs/developers/HowToStart.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+You need to first install Node Package Manager(`npm`). Please find `package.json` file in the root directory of this project. It has all the dependencies as long as the other major configuration of this project. In 'devDependencies' section, you can see that it includes @types/node and @types/vscode. 
+
+If you hit F12 on `import * as vscode from "vscode"` line, you can enter index.d.ts which has definitions of all vscode APIs you need. The documentation in comment is helpful enough sometimes, but to find more detailed description, go to [VSCode's official API page](https://code.visualstudio.com/api).
+
+### How to build the extension and debug?
+
+1. You must have `npm`(node package manager) installed.
+`npm` is a package manager like pip or apt.
+Likewise python virtual env, it manages packages directory by directory.
+
```
Comment (shs-park) [inline docs/developers/HowToStart.md:18]:
```shell
sudo apt install npm
```

File: docs/developers/HowToStart.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+2. Run `npm install`
+npm will prepare dependencies in `node_modules`.
+
```
Comment (shs-park) [inline docs/developers/HowToStart.md:21]: Move to the `ONE-vscode` directory and install the `node modules`.
```Shell
cd ONE-vscode
npm install
```

**Review by shs-park — APPROVED** [review]
Thanks for the kind documents for new starters.
I added some specific shell commands.

**Comment by shs-park** [comment]
> Signed-off-by: Dayoung Lee [dayoung.lee@samsung.com](mailto:dayoung.lee@samsung.com)

@dayo09, the signed-off prefix wording seems slightly different. Which one is correct?
Or, PR commit message doesn't need to follow `ONE-vscode-DCO-1.0-` prefix?

> ONE-vscode-DCO-1.0-Signed-off-by: Random O Developer <random@developer.example.org>

**Comment by dayo09** [comment]
@shs-park Thanks you for the comment! It's a mistake. I will fix it :-D (ONE-vscode-DCO* prefix is required)

File: docs/developers/HowToContribute.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+We use `prettier`.
+Your Pull Request will fail the CI if you don't follow this formatter's style. 
+It is included in our npm dependency, so additional installation of formatter is not needed.
+
+```bash
+cd ONE-vscode
+bash infra/format
+```
+
+
```
Comment (jyoungyun) [inline docs/developers/HowToContribute.md:36]:
```suggestion
```

**Comment by dayo09** [comment]
@jyoungyun PTAL :-D

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**